hail/python/test/hail/vds/test_vds.py

```diff
@@ -43,6 +43,7 @@ def test_validate():
             vds.variant_data).validate()
 
 
```

Comment:
also test `test_split_sparse_roundtrip` b/c split multi is the composition of multiple tricky code paths